src/components/MapView/MapViewTypes.ts

```diff
@@ -38,6 +38,9 @@ type PendingMapViewProps = {
 
     /** Style applied to PendingMapView */
     style?: StyleProp<ViewStyle>;
+
+    /** Whether the size of the route pending icon is smaller. */
+    isSmallerIcon?: boolean;
```

Comment:
I remember asking this before, can we pass the icon size number directly?

Reply:
Now we have only two icon sizes and wherever we want a smaller size we pass `isSmallerIcon` as true, I don't see any advantage of passing the icon size whenever we need a smaller one instead of flagging it as smaller icon via `isSmallerIcon`.

Reply:
Alright, that's fair.